Clockify.Net/ClockifyClient.cs

```diff
@@ -423,6 +423,21 @@ public Task<IRestResponse<TimeEntryDtoImpl>> CreateTimeEntryAsync(string workspa
             return _client.ExecutePostAsync<TimeEntryDtoImpl>(request);
         }
 
+        /// <summary>
+        /// Add a new time entry to workspace for another user. If end is not sent in request means that stopwatch mode is active, otherwise time entry is manually added.
+        /// </summary>
+        public Task<IRestResponse<TimeEntryDtoImpl>> CreateTimeEntryForAnotherUserAsync(string workspaceId, TimeEntryRequest timeEntryRequest)
```

Comment:
Almost done. Great job. I think it will be more verbose if `CreateTimeEntryForAnotherUserAsync` would look like this

```
CreateTimeEntryForAnotherUserAsync(string workspaceId, string userId, TimeEntryRequest timeEntryRequest)
```

Adding `UserId` will make this cleaner. Because it will state that `UserId` is necessary for the method and not hidden in 
`TimeEntryRequest`

Add that small change and I'll merge git and release a new version on Nuget!

Reply:
I will do that. Had the same thought my self, but was "disturbed" by the userid on TimeEntryRequest. If it is not used for anything else you could consider removing the userid from TimeEntryRequest